icicle/appUtils/large_ntt/thread_ntt.cu

```diff
@@ -0,0 +1,628 @@
+#ifndef T_NTT
+#define T_NTT
+#pragma once
+
+#include <stdio.h>
+#include <stdint.h>
+#include "curves/curve_config.cuh"
+
+struct stage_metadata {
+  uint32_t th_stride;
+  uint32_t ntt_block_size;
+  uint32_t ntt_block_id;
+  uint32_t ntt_inp_id;
+};
+
+uint32_t constexpr STAGE_SIZES_HOST[31][5] = {
```

Comment:
misleading formatting..

Reply:
I removed the comments. I think it's clear now but alternatively can add comment per case and have many lines instead.